docs/source/models/causes/alzheimers/presymptomatic_and_mci_gbd_2021/index.rst

```diff
@@ -0,0 +1,187 @@
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2021_cause_alzheimers_presymptomatic_mci:
+
+==================================================================
+Alzheimer's disease  with presymptomatic and MCI stages (GBD 2021)
+==================================================================
+
+.. contents::
+  :local:
+
+.. list-table:: Abbreviations
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+  * - AD
+    - Alzheimer's Disease
+  * - BBBM
+    - Blood-Based Biomarker
+  * - CSU
+    - Client Services Unit
+  * - MCI
+    - Mild Cognitive Impairment
+  * - YLD
+    - Years Lived with Disability
+  * - YLL
+    - Years of Life Lost
+
+Disease Overview
+++++++++++++++++
+
+GBD 2021 Modeling Strategy
+++++++++++++++++++++++++++
+
+Restrictions
+------------
+
+The following table describes any restrictions in GBD 2021 on the
+effects of the cause "Alzheimer's disease and other dementias" (such as
+being only fatal or only nonfatal), as well as restrictions on the ages
+and sexes to which the cause applies. We also list the implied age
+restriction on YLDs for the MCI-AD state of the cause model below.
+
+.. list-table:: GBD 2021 Cause Restrictions
+  :widths: 15 15 20
+  :header-rows: 1
+
+  * - Restriction Type
+    - Value
+    - Notes
+  * - Male only
+    - False
+    -
+  * - Female only
+    - False
+    -
+  * - YLL only
+    - False
+    -
+  * - YLD only
+    - False
+    -
+  * - YLL age group start
+    - 40 to 44
+    - age_group_id = 13
+  * - YLL age group end
+    - 95 plus
+    - age_group_id = 235
+  * - YLD age group start
+    - * 40 to 44 for AD cause state
+      * 30 to 34 for MCI-AD cause state
+    - * Restriction to age_group_id = 13 (40 to 44) for AD cause state
+        is from GBD
+      * Restriction to age_group_id = 11 (30 to 34) for MCI-AD cause
+        state is because we will be adding simulants at most 7 years
+        before AD incidence (so 40 - 7 = 33, in the 30-34 age group)
```

Comment:
@SylLutze @aflaxman Is this clear enough?

Reply:
I think so, but will there even be some low probability of MCI-AD being more than 7 years?   And what about pre-symptomatic?

Reply:
Pre-symptomatic doesn't have disability weight so no YLDs. 

And we will be adding sims this amount of time in advance. If they proceed unusually quickly or slowly after doesn't matter, we will capture it here anyways. 

I think this looks good!